pkg/cli/check_diff.go

```diff
@@ -1,22 +1,36 @@
 package cli
 
 import (
+	"fmt"
 	"os"
 	"path/filepath"
+	"runtime"
 
 	"github.com/spf13/cobra"
 	"github.com/wolfi-dev/wolfictl/pkg/checks"
 )
 
 func Diff() *cobra.Command {
 	o := checks.NewDiff()
+	var apkIndexURL string
 	cmd := &cobra.Command{
 		Use:               "diff",
 		DisableAutoGenTag: true,
 		SilenceUsage:      true,
 		SilenceErrors:     true,
 		Short:             "Create a diff comparing proposed apk changes following a melange build, to the latest available in an APKINDEX",
 		RunE: func(cmd *cobra.Command, _ []string) error {
+			arch := ""
+			switch runtime.GOARCH {
+			case "amd64":
+				arch = "x86_64"
+			case "arm64":
+				arch = "aarch64"
+			default:
+				return fmt.Errorf("architecture %s not supported", runtime.GOARCH)
+			}
```

Comment:
FWIW apko has `ParseArchitecture` and `ToAPK` to do this for more archs: https://pkg.go.dev/chainguard.dev/apko@v0.7.3/pkg/build/types#Architecture

If it's helpful we could probably do `types.CurrentArchitecture` that looks at `runtime.GOARCH`.